<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Human Entry/Exit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>431</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter</string>
    </property>
    <widget class="QLineEdit" name="bgf">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>220</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bff">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>16</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Background (frames)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buffer(frames)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius(R)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="r">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance(D)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="d">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Constraint In(Cin)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>24</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Constraint Out(Cout)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cout">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance Camera</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dis">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tCID">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Display Function</string>
    </property>
    <widget class="QCheckBox" name="chkBGM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>BGM</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkFF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>FF</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MF Line</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkEnt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkD">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>175</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Pause/Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>175</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>175</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Video</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="frameNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame#</string>
     </property>
    </widget>
    <widget class="QLabel" name="BGLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>BG</string>
     </property>
    </widget>
    <widget class="QLabel" name="BFLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>BF</string>
     </property>
    </widget>
    <widget class="QLabel" name="FFLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>201</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Foreground Feature</string>
     </property>
    </widget>
    <widget class="QLabel" name="CurrFLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>201</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Feature</string>
     </property>
    </widget>
    <widget class="QLabel" name="FPSLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
    <widget class="QLabel" name="SFLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>SFF</string>
     </property>
    </widget>
    <widget class="QLabel" name="MFLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>MFF</string>
     </property>
    </widget>
    <widget class="QLabel" name="EntLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
    <widget class="QLabel" name="ExtLabel_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>STATUS </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QLineEdit" name="tTC">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0,1,2,3,4,5,6,7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbTC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Testing case</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb3DC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Distance Calculate</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbFM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Feature Matching</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbLDS">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Default Setting</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbDis">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cam">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>175</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCDHEE">
    <property name="title">
     <string>Human Entry/Exit</string>
    </property>
   </widget>
   <addaction name="menuCDHEE"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
